core/src/main/resources/hudson/PluginManager/available.jelly

```diff
@@ -43,13 +43,34 @@ THE SOFTWARE.
                         value="${request.getParameter('filter')}" />
         </div>
         <div class="jenkins-app-bar__controls">
+          <l:isAdmin>
+            <div class="jenkins-split-button">
+              <button id="button-install" form="form" type="submit" name="dynamicLoad" class="jenkins-button jenkins-button--primary">
+                <div class="jenkins-dropdown__item__icon">
+                  <l:icon src="symbol-download" />
```

Comment:
using download icon to me is confusing as I (the user) am not downloading anything to my machine.
Rather Jenkins is installing - which is normally an arrow pointing right or the like.